tests/test_RestartSerialization.cpp

```diff
@@ -96,6 +97,21 @@ namespace Opm {
 }
 TEST_FOR_TYPE_NAMED(BVec, BlockVectorWrapper)
 
+BOOST_AUTO_TEST_CASE(SingleWellState)
+{
+    Opm::ParallelWellInfo dummy;
+    auto data_out = Opm::SingleWellState::serializationTestObject(dummy);
+    Opm::Serialization::MemPacker packer;
+    Opm::Serializer ser(packer);
+    ser.pack(data_out);
+    const size_t pos1 = ser.position();
+    decltype(data_out) data_in("", dummy, false, 0.0, {}, Opm::PhaseUsage{}, 0.0);
```

Comment:
Passing in a non-empty perf_input here would demonstrate my point above.

Reply:
To sum up my initial reluctance here: I think it is a bit dangerous to create a serializeOp that is essentially only useful for that one particular use (restarting). However, if we ever have another use case here (passing the state to another MPI rank for example), we would need to refactor the class and construction anyway, so I think it is acceptable. Will merge.